### ISSUE TITLE
[Request] UI and Control Improvement 

### ISSUE THREAD

**Issue by zero77** [dillbyrne/random-agent-spoofer#556]
Can you please add a menu that allows you to manually select what is being blocked for each website, for example like uMatrix in the below image.

But instead of css, image, etc can you add the items from the header and standard options tab and then show what is being blocked and let users decide if they want to or not.

Thank you

![untitled](https://cloud.githubusercontent.com/assets/16563034/26100393/a77a428a-3a25-11e7-99fe-2450896d0438.png)

**Comment by SMed79** [comment]
Whitelist is already supported

![capture du 2017-05-16 14-59-34](https://cloud.githubusercontent.com/assets/1873139/26109817/540ab616-3a48-11e7-92e8-4b0167261392.png)

**Comment by zero77** [comment]
Yes, but that white lists the whole domain although, what I'm suggesting would white lists specific things per domain.
